man/kbmadm.1m

```diff
@@ -0,0 +1,249 @@
+.\"
```

Comment:
Please add:

- A reference to appropriate documentation that explains what a template is and how one may construct one.
- A reference to documentation explaining plugins.

Reply:
The only documentation we have for either are the internal wiki pages and GitHub readmes -- do we need to get that out to docs.joyent.com before we integrate this?

Reply:
For now, point at a section of the appropriate RFD or pivy docs on github.  As the official Triton docs are updated, we can adjust the URL.  The key thing is that I don't want to inhibit early tire-kickers in the event that documentation updates are delayed.